man.Rmd

```diff
@@ -217,10 +284,24 @@ It's a little extra work keeping it all up to date, but the extra effort is ofte
 #' `grepl(pattern, string)`.
 ```
 
-If you want to use multiple paragraphs or a bulleted list, you can use the explicit `@description` tag[^man-4].
-Here's an example from `stringr::str_like()`, which mimics the `LIKE` operator from SQL:
+If you want more than one paragraph, you must use an explicit `@description` tag to prevent the second (and subsequent) paragraphs from being turned into the `@details`.
+Here's a two-paragraph `@description` from `stringr::str_view()`:
```

Comment:
I chose `str_view()` to get a straight-up example of 2 paragraphs. We'll show `str_like()` and a bullet list _after_ this.

I think it's worth being super explicit about the risk of accidentally creating a details section, when you really just wanted a two-paragraph description.